src/main/kotlin/no/nav/tilleggsstonader/sak/behandlingsflyt/FerdigstillBehandlingSteg.kt

```diff
@@ -23,7 +26,8 @@ class FerdigstillBehandlingSteg(
         logger.info("Ferdigstiller behandling=${saksbehandling.id}")
         behandlingService.oppdaterStatusPåBehandling(saksbehandling.id, BehandlingStatus.FERDIGSTILT)
 
-        taskService.save(InterntVedtakTask.lagTask(saksbehandling.id))
+        val taskVersjon = if (unleashService.isEnabled(Toggle.BRUK_INTERNT_VEDTAK_V2)) "2" else "1"
+        taskService.save(InterntVedtakTask.lagTask(saksbehandling.id, taskVersjon))
```

Comment:
Hvis man hadde haft behov for dette kunde man latt InterntVedtakService håndtere versjoneringen

Reply:
Tanken bak versjoneringen på tasken var bare å kunne gjøre debugging enklere, hvis det skulle bli tvil om hvilken kode som ble kjørt i tasken.